packages/cli/lang/en.lyaml

```diff
@@ -1036,6 +1036,17 @@ en:
               linkText: "HubSpot's sample projects"
               url: "https://developers.hubspot.com/docs/platform/sample-projects?utm_source=cli&utm_content=project_create_whats_next"
               message: "See {{ link }}"
+        git:
```

Comment:
I ported some of the git-related logging utils back into here, so I had to also add the lang keys for them